packages/web-components/@groupby/elements-sayt/test/unit/sayt.test.ts

```diff
@@ -1085,4 +1095,50 @@ describe('Sayt Component', () => {
       expect(result).to.be.an.instanceof(TemplateResult);
     });
   });
+
+  describe('updateSearchboxInputTerm()', () => {
+    let requestUpdateSearchTerm;
+
+    beforeEach(() => {
+      requestUpdateSearchTerm = spy();
+      stub(sayt, 'querySelector')
+        .withArgs('[data-gbe-ref="autocomplete"]')
+        .returns({ requestUpdateSearchTerm });
+    });
+
+    it('should call the requestUpdateSearchTerm method on autocomplete when enter is pressed and the event target is within the searchbox component', () => {
+      stub(sayt, 'nodeInSearchbox').returns(true);
+
+      sayt.updateSearchboxInputTerm({ key: 'Enter' });
+
+      expect(requestUpdateSearchTerm).to.be.called;
+    });
+
+    it('should not call the requestUpdateSearchTerm method on autocomplete if a key other than enter is pressed', () => {
+      stub(sayt, 'nodeInSearchbox').returns(true);
+
+      sayt.updateSearchboxInputTerm({ key: 'Z' });
+
+      expect(requestUpdateSearchTerm).to.not.be.called;
+    });
+
+    it('should not call the requestUpdateSearchTerm method on autocomplete if the event node is not a child of searchbox', () => {
+      stub(sayt, 'nodeInSearchbox').returns(false);
+
+      sayt.updateSearchboxInputTerm({ key: 'Enter' });
+
+      expect(requestUpdateSearchTerm).to.not.be.called;
+    });
+
+    it('should not attempt to call the requestUpdateSearch term method on autocomplete if autocomplete does not exist', () => {
```

Comment:
This test also needs this stub I believe `stub(sayt, 'nodeInSearchbox').returns(true);`.